Automattic/wp-calypso
# HC: Fix hover and focus styles for jump to recent

Fixes https://github.com/Automattic/dotcom-forge/issues/10033

## Proposed Changes
| Before  | After |
| ------------- | ------------- |
| <img width="408" alt="Screenshot 2024-12-03 at 16 51 37" src="https://github.com/user-attachments/assets/efb309cd-cc8e-4d1a-950c-112175f72a5c">  | <img width="414" alt="Screenshot 2024-12-03 at 16 50 47" src="https://github.com/user-attachments/assets/b8837d13-0786-4ead-85e9-fbc812872797">  |



* Fix jump to recent hover and focus styles

## Why are these changes being made?


* The wpcom styles should be respected

## Testing Instructions
- Checkout this PR
- Change the admin color scheme for your user
- Open the help center
- Verify the jump to recent button styles for hover and focus
- Run `yarn dev --sync` and verify the same in wp-admin
